source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -1,198 +1,350 @@
+   increasing the number of virtual processors allocated to the virtual 
+   machine under Settings --> System --> Processor for increased 
+   performance.
 
-To install the VirtualBox kernel modules, here are the steps:
+.. _vbox-start-vm-and-lga-begin:
 
-#. Install the bundle that supports VirtualBox modules::
+Start the |CL| VM
+*****************
 
-     swupd bundle-add kernel-lts
+The |CL| VM can now be powered on and setup.
 
-#. Set a timeout in the bootmanager to shows a menu at boot time::
+General instructions for using a |VB| virtual machine are available on the 
+`VirtualBox manual section on Running a VM`_.
 
-     clr-boot-manager set-timeout 10
+#. Start the VM from the |VBM| by selecting the |CL| VM and clicking *Start*
 
-#. Update the bootloader entries with::
+   .. image:: ./figures/vbox/vbox-start-vm.png
+      :alt: Starting a VirtualBox VM
 
-     clr-boot-manager update
+#. |CL| will boot and prompt for login.
 
-#. Reboot your system with::
+    - Enter **root** for the username. 
 
-     reboot
+#. You will be immediately prompted to set a new password for the **root** 
+   user. Reference :ref:`security` for more information about |CL| security 
+   concepts.
 
-   and choose **clear-linux-lts-4.14.XX-YYY** kernel version.
+   .. image:: ./figures/vbox/vbox-cl-first-login.png
+      :alt: Initial login to Clear Linux OS on a VirtualBox VM
 
-#. (*Optional*) Unset timeout to boot directly to LTS version::
 
-     clr-boot-manager set-timeout 0
 
-#. (*Mandatory*) Update bootmanger to always use LTS version::
+Install |VB| Linux Guest Additions 
+==================================
 
-     clr-boot-manager update
+The |VB| Linux Guest Additions provide drivers for full compatibility and 
+functionality. 
 
-Install Guest Additions
-=======================
+|CL| provides |VB| guest drivers and an install script in the **kernel-lts** 
+(Long Term Support) by |CL|.
 
-The kernel modules are shipped with the ``kernel-lts`` bundle. Insert Guest 
-Additions CD image using *Devices* menu you'll need to install the *user* 
-Linux Guest Additions. To install the VirtualBox Guest Additions, 
-follow these steps:
 
+#. Validate the installed kernel is **kernel-lts** by checking the output 
+   of the :command:`uname -r` command. It should end in **.lts**.
+
+   .. code-block:: bash
+
+      uname -r
+      4.<VERSION>.lts
+
+   If the running kernel is not **lts**: install the LTS kernel manually, update the bootloader, and check again:
+
+   .. code-block:: bash
 
-#. Insert Guest Additions CD image using *Devices* menu 
+      swupd bundle-add kernel-lts
+      clr-boot-manager set-kernel $(basename $(realpath /usr/lib/kernel/default-lts))
+      clr-boot-manager update
+      reboot
 
-   .. image:: ./figures/vbox-cd.png  
+#. Remove any non-LTS kernels to simplify and avoid conflicts:
+
+   .. code-block:: bash
+
+      swupd bundle-list | grep kernel
+      swupd bundle-remove <NON-LTS-KERNEL>
```

Comment:
Careful here... kernel-install is a bundle, too. This is clunky, but might be more precise:
`for type in $(sudo clr-boot-manager list-kernels | cut -d . -f 3 | sort -u); do sudo swupd bundle-list | grep "^kernel-${type}\$"; done`

Reply:
Maybe it's just safe to remove only kernel-native since we're guiding them at starting from scratch.

Reply:
Thanks for pointing this out. I'll clarify in the wording. I'd like to keep the commands simple and easy to digest for understanding purposes.